api-auto-registration/usage.hbs.md

```diff
@@ -114,4 +114,14 @@ spec:
         kind: HTTPProxy
         name: my-httpproxy
 
-```
\ No newline at end of file
+```
+
+## <a id='cors'></a>CORS
+
+The [CORS](https://fetch.spec.whatwg.org/#http-cors-protocol) protocol is used by the agent (usually a browser) to verify whether an API is used by the current origin. Your API must be configured to allow CORS Requests from TAP GUI. How you accomplish this varies based on the programming language and framework you are using. If you are using Spring you can get more information [here](https://spring.io/blog/2015/06/08/cors-support-in-spring-framework). If you are using something else please refer to it's documentation.
```

Comment:
Maybe add a sentence or two at the beginning of this paragraph on who might encounter this error. Like "If you want to use the "Try it out" feature for OpenAPI specs from the API Docs plugin, you will need to setup CORS to allow successful requests" or something like that

Reply:
changed